review04/passenger-service/src/main/java/cz/muni/fi/airportmanager/passengerservice/model/CreatePassengerDto.java

```diff
@@ -1,9 +1,8 @@
 package cz.muni.fi.airportmanager.passengerservice.model;
 
-public class Passenger {
-    public int id;
+public class CreatePassengerDto {
     public String firstName;
     public String lastName;
     public String email;
-    public int flightId;
+    public Long flightId;
```

Comment:
hopefully this will be aligned, and not required by students :)

Reply:
I think it's aligned across the whole 4-10 lectures